libr/core/cmd_print.c

```diff
@@ -4354,13 +4355,56 @@ static const char* bits_to_c_code_fmtstr(int bits) {
 	}
 }
 
+static void print_c_instructions(RPrint *p, ut64 addr, const ut8 *buf, int len, int ws) {
+	size_t i;
+
+	ws = R_MAX (1, R_MIN (ws, 8));
+	int bits = ws * 8;
+	const char *fmtstr = bits_to_c_code_fmtstr (bits);
+	len /= ws;
+
+	p->cb_printf ("#define _BUFFER_SIZE %d\n", len);
+	p->cb_printf ("const uint%d_t buffer[_BUFFER_SIZE] = {\n  ", bits);
+	p->interrupt = false;
+
+	int left = 0;
+	const int orig_align = p->coreb.cfggeti (p->coreb.core, "asm.cmt.col") - 40;
```

Comment:
same here. why do we need`cfggeti`? just directly use the API to get the var

Reply:
no, i cant